cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,84 @@
+import { generateUser } from '../support/generate';
 /// <reference types='cypress' />
 
+const user = generateUser();
+
+const rowsChoose = user.rows[Math.floor(Math.random() * user.rows.length)];
+
+const worker = 'Cierra';
+
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('pagination testing', () => {
+    cy.get('.-pageInfo').should('contain', 'Page');
+
+    cy.get('select').select(rowsChoose);
+
+    for (let i = 0; i < parseInt(rowsChoose.split(' ')[0]); i++) {
+      cy.get(`:nth-child(${i + 1})`).should('exist');
+    }
+  });
+
+  it('should able to add and delete new worker', () => {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+
+    cy.get('#submit').click();
+
+    cy.get('[id="delete-record-4"]').click();
+
+    cy.get(':nth-child(4)').should('not.contain', user.firstName);
+  });
+
+  it('should be able to delete all workers', () => {
+    for (let i = 0; i < 3; i++) {
+      cy.get(`[id="delete-record-${i + 1}"]`).click();
+    }
+
+    cy.get(':nth-child(4)').should('not.contain', user.firstName);
+
+    cy.get('div.noData').should('contain', 'No rows found');
```

Comment:
```suggestion
    cy.get('div.noData').should('contain', 'No rows found');
```
your test fails on this line

Reply:
fixed